runtime-config.md

```diff
@@ -120,3 +120,228 @@ Only [mounts from the portable config](config.md#mount-points) will be mounted.
 ```
 
 See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [SetVolumeMountPoint](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365561(v=vs.85).aspx) in Windows.
+
+## Configuration Schema Example
+
+Here is a full exmample `runtime.json` contents for reference (generated with `runc spec`).
+
+```json
+{
+	"mounts": {
+		"cgroup": {
+			"type": "cgroup",
+			"source": "cgroup",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev",
+				"relatime",
+				"ro"
+			]
+		},
+		"dev": {
+			"type": "tmpfs",
+			"source": "tmpfs",
+			"options": [
+				"nosuid",
+				"strictatime",
+				"mode=755",
+				"size=65536k"
+			]
+		},
+		"devpts": {
+			"type": "devpts",
+			"source": "devpts",
+			"options": [
+				"nosuid",
+				"noexec",
+				"newinstance",
+				"ptmxmode=0666",
+				"mode=0620",
+				"gid=5"
+			]
+		},
+		"mqueue": {
+			"type": "mqueue",
+			"source": "mqueue",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev"
+			]
+		},
+		"proc": {
+			"type": "proc",
+			"source": "proc",
+			"options": null
+		},
+		"shm": {
+			"type": "tmpfs",
+			"source": "shm",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev",
+				"mode=1777",
+				"size=65536k"
+			]
+		},
+		"sysfs": {
+			"type": "sysfs",
+			"source": "sysfs",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev"
+			]
+		}
+	},
+	"hooks": {
+		"prestart": null,
+		"poststart": null,
+		"poststop": null
```

Comment:
[Another][1] location where I think we should fill in something useful.  If we remove the whole section, there wouldn't be any context here to anchor the [hook specs][2].

[1]: https://github.com/opencontainers/specs/pull/276/files#r48075833
[2]: https://github.com/opencontainers/specs/blob/e79365a749f86254dec96609caff2a8acd15f689/runtime-config.md#poststop